### PR TITLE
[DOCS] Adds tip for elastic built-in user

### LINKED ISSUE #29798: [DOCS] Discourage use of elastic superuser
*Original comment by @tvernum:*

New usershave a tendency to use the `elastic` user for lots of things that don't actually need superuser privileges.

While we're never going to stop that from happening, we should be more explicit in the docs that this is not recommended, and they ought to create their own users with the minimum necessary privileges.

### PR DESCRIPTION
**Pull request by lcawl**
Fixes https://github.com/elastic/elasticsearch/issues/29798

Preview: http://elasticsearch_51891.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/built-in-users.html

### CODE REVIEW

**Comment by lcawl** [comment]
@elasticmachine update branch

File: x-pack/docs/en/security/authentication/built-in-users.asciidoc [hunk 1]
```diff
@@ -16,6 +16,10 @@ used to <<set-built-in-user-passwords,set all of the built-in user passwords>>.
 storing monitoring information in {es}. It has the `remote_monitoring_agent` and 
 `remote_monitoring_collector` built-in roles. 
 
+TIP: The built-in users serve a specific purpose and are not intended for every
+day use. In particular, do not use the `elastic` superuser unless full access to
```
Comment (tvernum) [inline x-pack/docs/en/security/authentication/built-in-users.asciidoc]: I worry about someone interpreting "not intended for every day use" to mean that they shouldn't use the `kibana` user in their `kibana.yml`.
I don't have a better suggestion though.

Comment (lcawl) [reply x-pack/docs/en/security/authentication/built-in-users.asciidoc]: Good point, I've switched to "general use" instead of "every day use".

**Comment by lcawl** [comment]
@elasticmachine update branch

**Review by tvernum — APPROVED** [review]

**Pull request merged**